cnxarchive/sql/search/author.part.sql

```diff
@@ -17,7 +17,7 @@ WHERE
    OR
    u.last_name ~* req(%({0})s::text)
    OR
-   u.full_name ~* req(%({0})s::text)
+   u.full_name ~* regexp_replace(req(%({0})s::text), ' ', '.*.')
```

Comment:
Yeah, I guess that makes sense. Come to think about it, I can see where a global replace would be better - if someone has a duel first or last name (also, just in case). I'll update and issue another PR.